docs/infrastructure_configuration_variables.md

```diff
@@ -0,0 +1,39 @@
+# Infrastructure Configuration Variables
+
+The infrastructure configuration variables are used to configure the infrastructure for the cluster. These variables are located in  infrastructure_configuration.yaml file. 
+
+| Variable | Description | Default | Provider |
+| -------- | ----------- | ------- | -------- |
+| `provider` | The provider to use for the infrastructure. | `AWS` | `all` |
+| `hive_cluster_name` | The name of the cluster. | `rhods{provider}` | `all` |
+| `hive_claim_name` | The name of the claim. | `rhods{provider}claim` | `all` |
+| `image_set` | The image set name to use for the cluster. | `rhods-openshift` | `all` |
+| `aws_region` | The AWS region to use for the cluster. | `us-east-1` | `aws` |
+| `worker_node_instance_type` | The instance type to use for the worker nodes. | `m5.xlarge` | `all` |
+| `worker_node_replicas` | The number of worker nodes to create. | `2` | `all` |
+| `master_node_instance_type` | The AWS instance type to use for the master nodes. | `m5.xlarge` | `all` |
+| `master_node_replicas` | The number of master nodes to create. | `3` | `all` |
+| `aws_region` | The AWS region to use for the cluster. | `us-east-1` | `aws` |
+| `pull_secret` | The pull secret to use for the cluster. | `""` | `all` |
```

Comment:
what does it mean? Pull secret for pulling what exatcly?

Reply:
It is the cluster's default pull secret